docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/how-to-guides/how-to-document-glue-api-endpoints.md

```diff
@@ -100,28 +120,29 @@ class PaymentMethodsByCheckoutDataResourceRelationshipPlugin extends AbstractPlu
+{% endinfo_block %}
+
+The following table lists descriptions of the properties you can use in the annotations:
 
-|     |     |
+| PROPERTY | DESCRIPTION |
 | --- | --- |
-|     |     |
-| `deprecated` | The path will be marked deprecated. |
-| `responseAttributesClassName` | The transfer will be used as a representation of the response attributes. |
-| `isIdNullable` | The (JSON:API) response `id` will be null. |
-| `path` | The resource is available on the path. This property can be used if the type+id is not the path the resource uses. |
-| `summary` | Path’s summary. Text should briefly describe what the endpoint is doing. |
-| `parameters` | The list of parameters the endpoint is accepting. |
-| `isEmptyResponse` | A flag is to be used to mark endpoint that return empty responses. |
-| `responses` | A list responses the endpointcan respond with. The object must contain key-value pairs with HTTP codes as key, and desciption as value. |
+| `deprecated` | Path is marked deprecated. |
+| `responseAttributesClassName` | Transfer is used as a representation of the response attributes. |
+| `isIdNullable` | JSON:API response `id` is null. |
+| `path` | Resource is available on the path. This property can be used if the *type+id* is not the path the resource uses. |
+| `summary` | Path's summary that briefly describes what the endpoint does. |
+| `parameters` | List of parameters the endpoint accepts. |
+| `isEmptyResponse` | Fag used to mark endpoint that return empty responses. |
+| `responses` | List responses the endpoint can respond with. The object must contain key-value pairs with HTTP codes as key, and desciption as value. |
 
 ### Extending the behavior
 
 The following interfaces can be used to add more data to the generated documentation.
 
-1.  `Spryker\Glue\DocumentationGeneratorApiExtension\Dependency\PluginApiApplicationProviderPluginInterface` - adds a new application to which the documentation will get generated.
-    
-2.  `Spryker\Glue\DocumentationGeneratorApiExtension\Dependency\Plugin\ContextExpanderPluginInterface` - adds information to the documentation generation context.
-    
-3.  `Spryker\Glue\DocumentationGeneratorApiExtension\Dependency\Plugin\SchemaFormatterPluginInterface` - formats the part of the documentation, and must return an array of data ready for getting converted to YAML.
+* The `Spryker\Glue\DocumentationGeneratorApiExtension\Dependency\PluginApiApplicationProviderPluginInterface` interface: Adds a new application to which the documentation will get generated.
+* The `Spryker\Glue\DocumentationGeneratorApiExtension\Dependency\Plugin\ContextExpanderPluginInterface` interface: Adds information to the documentation generation context.
+* The `Spryker\Glue\DocumentationGeneratorApiExtension\Dependency\Plugin\SchemaFormatterPluginInterface` interface: Formats the part of the documentation and must return an array of data ready for getting converted to YAML.
```

Comment:
data that is ready for getting converted to YAML?

Reply:
This is the question for Ahmed as well